packages/driver/test/cypress/integration/commands/navigation_spec.coffee

```diff
@@ -440,6 +440,11 @@ describe "src/cy/commands/navigation", ->
         .visit("http://localhost:3500/fixtures/generic.html")
         .visit("http://localhost:3500/fixtures/dimensions.html")
 
+    it "can visit the same page with hashes", ->
```

Comment:
Whenever you're writing / adding tests that specifically fix an issue, add a comment above the test with a link to the github issue to help indicate why this test was added.